### PR TITLE
Manual monero sweep

### LINKED ISSUE #538: Allow manual sweeping for Monero Keys
Similar to the BItcoin Keys, it should be possible for the user to manually sweep the keys.

### PR DESCRIPTION
**Pull request by sedited**
Based on #540

### CODE REVIEW

File: src/farcasterd/runtime.rs [hunk 1]
```diff
@@ -1670,6 +1670,56 @@ impl Runtime {
                 )?;
             }
 
+            Request::SweepXmrAddress(sweep_xmr_address) => {
+                // remove once conversion is implemented in farcaster core Network type
```
Comment (TheCharlatan) [inline src/farcasterd/runtime.rs:1674]: Change comment to TODO

File: src/cli/opts.rs [hunk 2]
```diff
@@ -242,14 +242,23 @@ pub enum Command {
...
+    /// Attempts to sweep any funds on a given bitcoin funding address
+    #[display("sweep-bitcoin-address<{source_address} {destination_address}>")]
+    SweepBitcoinAddress {
         /// The source address to be swept.
         source_address: BtcAddress,
         /// The destination address receiving the coins.
         destination_address: BtcAddress,
     },
+
+    /// Attempts to sweep any funds on a given bitcoin funding address
```
[the remark this answers is not in the document]
Comment (TheCharlatan) [reply src/cli/opts.rs]: 7409a0f96b475e5d4dac51d7758588594423e739

**Review by zkao — APPROVED** [review]
nice functionality adding the keys extracted from the buy and refund txs to a database and makking them available for manually sweeping later

**Comment by TheCharlatan** [comment]
I'll investigate.

**Comment by TheCharlatan** [comment]
Rebased.

**Pull request merged**